.github/workflows/maven-checks.yml

```
@@ -46,5 +46,67 @@ jobs:
+        with:
+          name: presto-server
+          path: presto-server/target/presto-server-0.293.tar.gz
+      - name: Upload presto-cli executable
+        if: matrix.java == '8.0.442'
+        uses: actions/upload-artifact@v4
+        with:
+          name: presto-cli
+          path: presto-cli/target/presto-cli-0.293-executable.jar
       - name: Clean Maven Output
         run: ./mvnw clean -pl '!:presto-server,!:presto-cli,!presto-test-coverage'
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)